runtime-watcher/internal/handler.go

```diff
@@ -250,9 +250,13 @@ func (h *Handler) sendRequestToKcp(moduleName string, watched WatchedObject) str
 	}
 	defer resp.Body.Close()
 	responseBody, _ := io.ReadAll(resp.Body)
```

Comment:
```suggestion
	responseBody, err := io.ReadAll(resp.Body)
```

Reply:
I think I should take a rest after doctor appointment 😅